### ISSUE TITLE
Gear cannot run if no inputs in manifest

### ISSUE THREAD

**Issue by jenreiter** [scitran/core#928]
When running a gear that takes no inputs, the follow error occurs when trying to start the job:
**A validation error has occurred. Please review any errors above and resubmit the form.**

Below is the manifest of the gear that was causing this error.

```
{
  "label": "Session Archive",
  "name": "session-archive",
  "version": "0.0.1rc1",
  "description": "This gear uses the Flywheel Python SDK to locate all files within a given session (indicated by session ID given in the config file) on a user's Flywheel Instance. All files within the session are outputted as a single zip archive.",
  "inputs": {},
  "config": {
    "session_id": {
      "default": "",
      "type": "string",
      "description": "ID number of the session."
    },
    "api_key": {
      "default": "",
      "type": "string",
      "description": "The API key of the user."
    }
  },
  "author": "Jennifer Reiter",
  "maintainer": "Jennifer Reiter",
  "custom": {
    "docker-image": "flywheel/session-archive:v0.0.1",
    "gear-builder": {
        "image": "flywheel/session-archive"
        }
  },
  "license": "Other",
  "url": "",
  "source": "https://github.com/flywheel-apps/session-archive"
}
```

**Comment by kofalt** [comment]
Will be closed by #938.

**Comment by nagem** [comment]
Resolved by #938
